test/openst/03_direct_transfer.js

```diff
@@ -0,0 +1,63 @@
+// 'use strict';
+
+const { assert } = require('chai');
+const BN = require('bn.js');
+const { ContractInteract } = require('@openst/mosaic.js');
+const BrandedToken = require('@openst/brandedtoken.js');
+
+const { chainConfig, connection } = require('../shared');
+const OpenST = require('../../src/openst');
+
+describe('Direct Transfer', async () => {
+  it('Transfers tokens from sender to receiver', async () => {
+    const auxiliaryUtilityToken = chainConfig.utilitybrandedtokens[0].address;
+
+    const openst = new OpenST(chainConfig, connection);
+    const {
+      beneficiary,
+    } = await openst.createUserWallet(
+      auxiliaryUtilityToken,
+      connection.auxiliaryAccount.address,
+      1,
+      connection.auxiliaryAccount.address, // Comma separated session keys.
```

Comment:
The revised testing framework uses addresses from accounts that are created via an RPC call and thus are not available as full `Account` objects locally. So, the `connection.auxiliaryAccount` object is thin, containing only an `address` member. The result of this is that those account objects do not possess the extended EIP 712 and 1077 signing functions.

Depending on what and how items are added in `shared`, a session key may need to be created in the course of this test, rather than using `connection.auxiliaryAccount.address`: https://github.com/OpenST/JLP/pull/60/files#diff-5f10fab689d672cc6cbca69d503f2660R36

Reply:
Thank you for detailed reply.
Here using worker as the session key for reusability. Worker account is create 
and private key is logged to config.json. This helps create session account object from private key for signing. Please see the updated code. In JLP smoke tests we have the flexibility to reuse same account for multiple use cases 👍 .